library/Icingadb/Common/Macros.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+/* Icinga DB Web | (c) 2021 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Icingadb\Common;
+
+use Icinga\Application\Logger;
+use Icinga\Module\Icingadb\Model\Host;
+use Icinga\Module\Icingadb\Model\Service;
+
+trait Macros
+{
+    /**
+     * Known icinga macros
+     *
+     * @var array
+     */
+    protected $icingaMacros = [
```

Comment:
Is this really required at all? The supported macros are way more *flexible* any way, so to me this map is rather redundant, isn't it?

Reply:
I think so too. It was actually mainly intended to resolve the uppercase macros.